pkgs/development/compilers/fbc/default.nix

```diff
@@ -0,0 +1,79 @@
+{ stdenv
+, lib
+, fetchzip
+, gpm
+, libffi
+, libGL
+, libX11
+, libXext
+, libXpm
+, libXrandr
+, ncurses
+, OpenGL
+, fbc-mac-bin
+}:
+
+stdenv.mkDerivation rec {
+  pname = "fbc";
+  version = "1.09.0";
+
+  src = fetchzip {
+    url = "https://github.com/freebasic/fbc/releases/download/${version}/FreeBASIC-${version}-source-bootstrap.tar.xz";
```

Comment:
Not using `fetchFromGitHub` here because the compiler is written in FreeBASIC itself and needs to be bootstrapped by another fbc installation. Those tarballs provide pre-translated source code so it can be built with just a C compiler / assembler on *some* platforms.

`make bootstrap-minimal` builds a native `fbc` from those files, regular `make` re-translates & re-compiles the final `fbc` with the bootstrapped one.

Reply:
Added a short comment about it.